app/models/season.rb

```diff
@@ -24,7 +24,7 @@ def started?
 
   def year; name end
 
-  # switch_phase, in dev env only: enables developers to easily switch
+  # switch_phase, not available in deployment: enables developers to easily switch
   # between time dependent settings in views
```

Comment:
Technically, the action on the model instance is available anywhere. The controller action isn't ;)